taxonomies/ingredients.txt

```diff
@@ -14353,6 +14353,7 @@ nn:ghi, ghee
 pa:ਘਿਉ
 pl:ghi, ghee
 pt:ghi, ghee
+ro:ulei de unt
```

Comment:
Do you think that we could add "ghee" as a synonym as well?
```suggestion
ro:ulei de unt, ghee
```

Reply:
I rather keep them separate, just to keep the indian variant as its own entity. You could add it to butter fat.

Reply:
@alexgarel, 
I understood that @aleene suggests to keep only ghee in this block (indian variant) and more ulei de unt in other entry (butter fat)

Reply:
suggestion from @aleene has been applied (you can ignore the first comment of this thread)